src/config.ts

```
@@ -70,17 +76,28 @@ export async function validateInvocation(
 ) {
   const { config } = context.instance;
 
-  await sanitizeConfig(config); //mutate the config as needed
+  sanitizeConfig(config); //mutate the config as needed
   const apiClient = createAPIClient(config, context.logger);
   await apiClient.verifyAuthentication();
 }
 
-export async function sanitizeConfig(config: IntegrationConfig) {
-  config.githubAppPrivateKey = 'temp';
-  config.githubAppPrivateKey = await fetchPrivateKey({
-    privateKeyEnvLocalPathParam: 'GITHUB_APP_LOCAL_PRIVATE_KEY_PATH',
-    privateKeyEnvSsmParam: 'GITHUB_APP_PRIVATE_KEY', //this a hack. We need to define the real param.
-  });
+export function sanitizeConfig(config: IntegrationConfig) {
+  const localPath = process.env['GITHUB_APP_LOCAL_PRIVATE_KEY_PATH'];
+  if (localPath) {
+    let content;
+    try {
+      content = fs.readFileSync(localPath);
+    } catch (err) {
+      // basically not there
```

We should throw an error here as it is going to error at content.toString anyway